addons/resource.language.en_gb/resources/strings.po

```diff
@@ -3608,7 +3608,7 @@ msgid "Singles"
 msgstr ""
 
 msgctxt "#1051"
-msgid "Enter web address"
+msgid "Enter Web address"
```

Comment:
The Web portion is still capitalized so derivative words shall also be.

See 2.1 http://www.oxforddictionaries.com/definition/english/web?q=Web#web__8 for example

But... Again.. In any program such as Kodi, as long as the usage of the terminology is consistently used, in the end the discussion is moot.